client/models/resource-link.js

```diff
@@ -26,58 +26,76 @@ function resourceLink(data, feather) {
     feather = feather || f.catalog().getFeather("ResourceLink");
     let model = f.createModel(data, feather);
     let d = model.data;
-    
-    /// Expression to extract a fileName portion of a URL, considering hash and parameter
-    /// Based on: https://stackoverflow.com/questions/14473180/regex-to-get-a-filename-from-a-url  
+
+    // Expression to extract a fileName portion of a URL,
+    // considering hash and parameter
+    // Based on: https://
+    // stackoverflow.com/questions/14473180/regex-to-get-a-filename-from-a-url
```

Comment:
The citation could be moved to a separate location.  It looks a bit odd to see the URL broken up like it is.